adapters/AppLovin/AppLovinAdapter/GADMAdapterAppLovinConstants.h

```diff
@@ -0,0 +1,12 @@
+//
+//  Copyright © 2017 AppLovin. All rights reserved.
+//
+
+static NSString *const kGADMAdapterAppLovinErrorDomain = @"com.applovin.sdk.mediation.admob.errorDomain";
+static NSString *const kGADMAdapterAppLovinVersion = @"AdMob-3.0";
```

Comment:
Our version naming convention for mediation adapters is [Network SDK version].[adapter patch number]. Please see the  [Facebook](https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Facebook/CHANGELOG.md) or [Unity Ads](https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Unity/CHANGELOG.md) adapter CHANGELOGs for examples.

Since the current version of AppLovin SDK is 4.3.0, the adapter version needs to be 4.3.0.0.